test/LiquidityPool.t.sol

```diff
@@ -93,8 +95,8 @@ contract LiquidityPoolTest is TestSetup {
     // --- erc165 checks ---
     function testERC165Support(bytes4 unsupportedInterfaceId) public {
         bytes4 erc165 = 0x01ffc9a7;
-        bytes4 erc7540Deposit = 0xea446681;
-        bytes4 erc7540Redeem = 0x2e9dd5bd;
+        bytes4 erc7540Deposit = 0x1683f250;
+        bytes4 erc7540Redeem = 0x0899cb0b;
```

Comment:
Just curious. What are these hex representations of?

Reply:
They are the interface IDs. See https://eips.ethereum.org/EIPS/eip-165